src/FsAutoComplete.Core/SignatureFormatter.fs

```diff
@@ -291,6 +293,8 @@ module SignatureFormatter =
 
         if p.Type.IsFunctionType then
           $"({formatted})"
+        else if p.IsOptionalArg && formatted.StartsWith("option<", StringComparison.Ordinal) then // render optional args as "?ident: type"
+          formatted.Substring(7).TrimEnd('>')
```

Comment:
nit - instead of creating a bunch of intermediates here, can you do a span-slice from 7..^1 and create the string directly from that?